calendar/Package.md

```diff
@@ -0,0 +1,20 @@
+Connects to Microsoft Outlook Calendar from Ballerina
+
+### Package overview
+The `outlook.calendar` is a [Ballerina](https://ballerina.io/) connector for Outlook Calendar.
+This package provides the capability to easily create appointments & events, organize meetings, manage user's calendar etc.
+
+#### Compatibility
+|                                 | Version                         |
+|---------------------------------|---------------------------------|
+| Ballerina Language              | Ballerina Swan Lake Beta2       |
+| Microsoft Outlook Calendar API  | v1.0                            |
+
+
+### Report issues
+To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina Microsoft calendar repository](https://github.com/ballerina-platform/module-ballerinax-microsoft.outlook.calendar)
+
+### Useful links
```

Comment:
Use ## heading

Reply:
Changed /Thanks

Reply:
Corrected in Module. md as well.
/Thanks